### PR TITLE
Use ffmpeg additional params writing videoclip audio

### PR DESCRIPTION
**Pull request by biarne-a**
- [ ] If this is a bugfix, I have provided code that clearly demonstrates the problem and that works when used with this PR
- [ ] I have added a test to the test suite, if necessary
- [ ] I have properly documented new or changed features in the documention, or the docstrings
- [ ] I have properly documented unusual changes to the code in the comments around it
- [ ] I have made note of any breaking/backwards incompatible changes

### CODE REVIEW

File: moviepy/video/VideoClip.py [hunk 1]
```diff
@@ -323,6 +323,7 @@ def write_videofile(
                 bitrate=audio_bitrate,
                 write_logfile=write_logfile,
                 verbose=verbose,
+                ffmpeg_params=ffmpeg_params,
```
Comment (mgaitan) [inline moviepy/video/VideoClip.py:326]: I'm wondering if wouldn't be desirable to allow different parameters for audio and video

Comment (mgaitan) [reply moviepy/video/VideoClip.py:326]: I mean, add an argument `audio_ffmpeg_params=None` to the method and pass that value here. It'd allow more granularity on the custom ffmpeg behaviour, right?

Comment (mondeja) [reply moviepy/video/VideoClip.py:326]: Yes, definetively.

**Comment by mondeja** [comment]
Closing this in favour of #1493

**Pull request closed without merging**